src/main/java/com/sprint/mission/discodeit/controller/ReadStatusController.java

```diff
@@ -0,0 +1,48 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.request.ReadStatusCreateRequest;
+import com.sprint.mission.discodeit.dto.request.ReadStatusUpdateRequest;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Controller
+@ResponseBody
+@RequestMapping("/api/readStatuses")
```

Comment:
여기도 하이픈으로 바꿔주면 좋을 것 같네요.  
```suggestion
@RequestMapping("/api/read-statuses")
```

Reply:
감사합니다 !